assets/js/main.js

```diff
@@ -385,6 +385,26 @@ function search() {
                 e.preventDefault();
                 $(`#search-element-${searchSelectionId}`).focus();
                 break;
+            case 'ArrowLeft':
+                //go back to top search bar
+                searchSelectionId = -1
+                if (searchSelectionId === -1) {
```

Comment:
Hmm I realised the if statement is redundant? Since `searchSelectionId` is already set to `-1` it'll run the if statement all the time, so may as well run the code outside the if statement